html-lists/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Lists</title>
+</head>
+
+<body>
+  <h1>HTML Lists</h1>
+  <h2>Ordered List Example</h2>
+  <ol>
+    <li>Learn to code.</li>
+    <li>Build Projects.</li>
+    <li>Get a job.</li>
+  </ol>
+  <h2>Unordered List Example</h2>
+  <li>HTML</li>
```

Comment:
you are missing the `ul` element for the `li`s. make sure to include that element!

Reply:
![image](https://user-images.githubusercontent.com/90783131/151103963-a67ce2c1-b708-4fbd-a3df-505e81d3cddb.png)